### PR TITLE
Added new ppc64le Dockerfile

### PR DESCRIPTION
**Pull request by cwesMills**
Added Dockerfile to create ubuntu16 containers on ppc64le.

Signed-off-by: Colton Mills <millscolt3@gmail.com>

### CODE REVIEW

**Comment by cwesMills** [comment]
I need to update JDK URLs before this can be merged

File: buildenv/jenkins/docker-slaves/ppc64le/ubuntu16/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    software-properties-common \
+    python-software-properties \
+  && add-apt-repository ppa:ubuntu-toolchain-r/test \
+  && apt-get update \
+  && apt-get install -qq -y --no-install-recommends \
+    ant \
+    ant-contrib \
+    autoconf \
+    build-essential \
+    cmake \
```
Comment (AdamBrousseau) [inline buildenv/jenkins/docker-slaves/ppc64le/ubuntu16/Dockerfile]: Did you try removing cmake?

File: buildenv/jenkins/docker-slaves/ppc64le/ubuntu16/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    cpio \
```
Comment (AdamBrousseau) [inline buildenv/jenkins/docker-slaves/ppc64le/ubuntu16/Dockerfile]: Do we know what cpio is used for?

File: buildenv/jenkins/docker-slaves/ppc64le/ubuntu16/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    curl \
+    file \
```
Comment (AdamBrousseau) [inline buildenv/jenkins/docker-slaves/ppc64le/ubuntu16/Dockerfile]: Is file included in the default image? Do we need it?

File: buildenv/jenkins/docker-slaves/ppc64le/ubuntu16/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    g++-4.8 \
+    gcc-4.8 \
+    git \
+    git-core \
```
Comment (AdamBrousseau) [inline buildenv/jenkins/docker-slaves/ppc64le/ubuntu16/Dockerfile]: Not sure about this one?
https://askubuntu.com/questions/5930/what-are-the-differences-between-the-git-and-git-core-packages

**Pull request merged**